traefik/templates/service.yaml

```diff
@@ -1,9 +1,22 @@
-{{- if .Values.service.enabled -}}
+{{- $services := .Values.service.additionalServices -}}
+{{- $services = set $services "default" (omit .Values.service "additionalServices") }}
+
+{{- range $name, $service := $services -}}
+{{- if $service.enabled -}}
+
+{{- $fullname := printf "%s-%s" (include "traefik.fullname" $) $name }}
+{{- if eq $name "default" -}}
+{{- $fullname = include "traefik.fullname" $ }}
+{{- end -}}
+
+{{- if ge (len $fullname) 60 }} # 64 - 4 (udp-postfix) = 60
+  {{- fail "ERROR: Cannot create a service whose full name contains more than 60 characters" }}
+{{- end }}
```

Comment:
Wdyt about moving this block in a `traefik.servicename` helper function ?

Reply:
Yeah that makes sense. Just pushed a commit to add this. Things are getting a bit complicated with all the dicts passing roots and other arguments involved, but it should be fine I think.